### DISCUSSION TITLE
OAuth route unavailable

### DISCUSSION THREAD

**Discussion by Wuk-jvi** [okd-project/okd#1081] [Q&A]
Hi,

I have installed new OKD 4.9.0-0.okd-2022-01-14-230113 cluster. If I use external HA proxy as load balancer (without TLS termination) everything works as expected. When I switch to F5 LB (DNS record change) with TLS termination I get error on authentication cluster operator:
`OAuthServerRouteEndpointAccessibleControllerAvailable: "https://oauth-openshift.apps.okd.testdomain.local/healthz" returned "503 Service Unavailable"`
F5 logs does not show anything useful just 503 response.
Can someone point me where I should look at the logs on OKD install to see what is happening.

**Comment by rvanderp3** [comment]
Are you forwarding SNI headers in the f5?  If not, the router won't know what service should receive the request and will return a 503.

**Reply by itewk** [reply]
@rvanderp3 do you  know how to setup the f5 to properly forward SNI headers in this scenario? Been trying to find the right doc and have come up short.
